lighthouse-core/lib/page-functions.js

```diff
@@ -107,12 +107,20 @@ function getElementsInDocument(selector) {
 /**
  * Gets the opening tag text of the given node.
  * @param {Element} element
+ * @param {Array<string>=} ignoreAttrs Optionally provides an array of attribute tags to not include in HTML the snippet
```

Comment:
I meant up in the function description, but down by the param is fine too. The word order got a little scrambled here...maybe "An optional array of attribute tags to not include in the HTML snippet."?